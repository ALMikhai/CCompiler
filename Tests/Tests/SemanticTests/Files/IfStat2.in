int f() {
    if (1) {
        int a = 2;
    }
    else 2 * 3;
}