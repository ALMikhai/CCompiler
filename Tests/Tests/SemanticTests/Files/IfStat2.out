\-TRANSLATIONUNIT
  \-FUNC
    |-DECLSPEC
    | |-INT
    |-INITDECLARATOR
    | \-DECLARATOR
    |   \-FUNC
    |     |-f
    |     \-EMPTY
    \-COMPOUNDSTAT
      \-STATLIST
        \-IF
          |-1
          |-COMPOUNDSTAT
          | |-DECLLIST
          | | \-DECL
          | |   |-DECLSPEC
          | |   | |-INT
          | |   \-INITDECLARATORLIST
          | |     \-INITDECLARATOR
          | |       |-DECLARATOR
          | |       | \-a
          | |       \-2
          \-MULT
            |-2
            \-3

Structs { }
Symbols {
	FUNC returning INT
	Arguments{ }
	Structs { }
	Symbols {
		Nested block {
			Structs { }
			Symbols {
				INT :: a
			}
		}
	} :: f
}
